packages/toast/plugin.js

```diff
@@ -1,8 +1,19 @@
 import Vue from 'vue'
 import Toasted from 'vue-toasted'
 
-Vue.use(Toasted, <%= serialize(options) %>)
+const opts = <%= serialize(options) %>
+var globals = []
```

Comment:
Can you please separate register from options at module level?

Reply:
So instead of register being inside the toast {} options they should be another option like register_toasts [] ?

Reply:
I mean a code like this in `index.js`:

```js
module.exports = function nuxtToast (moduleOptions) {
  const { register, ...toastOptions } = Object.assign({}, this.options.toast, moduleOptions)

  // Register plugin
  this.addPlugin({
    src: path.resolve(__dirname, 'plugin.js'),
    ssr: false,
    fileName: 'toast.js',
    options: {
       register,
       toastOptions
    }
  })
}
```

Reply:
Thanks, much cleaner.. works well. I updated the readme.